### DISCUSSION TITLE
Submit button in Newsletter block has limited settings

### DISCUSSION THREAD

**Discussion by katerby** [publishpress/publishpress-blocks#1199] [Feature Requests]
**Describe the bug**
That might be a area for improvement. In the submit button setting didnt find how to adjust size of the button, how to change location of the button in the block

**To Reproduce**
1. add Newsletter block
2. Go to submit button setting
3. See quite limited settings are allowed  

**Expected behavior**
Would be nice to be able  to adjust size of Submit button, change location of the button.

**Comment by htmgarcia** [comment]
Good point, @katerby! Thanks.
